README.md

```diff
@@ -16,8 +16,71 @@ This repository contains both a library and a standalone binary, named **crg**.
 CRGis shipped in two forms for maximum flexiblity:
 
 * Standalone executable `crg`
-The standalone executable will talk to your blockchain's API and serve as an API adapter, making it compatible with the Coinbase Rosetta blockchain API interface
+The standalone executable will talk to your blockchain's API and serve as an API adapter, making it compatible with the Coinbase Rosetta blockchain API interface.
 
+**Quick Test**:
+
+This assumes that you're using a "fresh" Ubuntu or Debian virtual machine.  
+
+You'll need [Go](https://golang.org) and nodejs 14.  We recommend that you use [NVM](https://github.com/nvm-sh/nvm) to install nodejs.  
+
+Install NVM and Go
+```bash
+# NVM
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash
+export NVM_DIR="$HOME/.nvm"
+[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"  # This loads nvm
+[ -s "$NVM_DIR/bash_completion" ] && \. "$NVM_DIR/bash_completion"  # This loads nvm bash_completion
+
+# Go
+wget https://golang.org/dl/go1.15.2.linux-amd64.tar.gz
+tar -C /usr/local -xzf go1.15.2.linux-amd64.tar.gz
+export PATH=$PATH:/usr/local/go/bin
+```
+
+Get a simple starport chain running in another session
+```bash
+sudo apt install -y screen git build-essential
+screen -S starport
+npm i -g @tendermint/starport
+starport app github.com/yourname/clay
+cd clay
+starport serve --verbose
+```
+
+Hold `ctrl` and press A, then D. You'll leave the screen session and the blockchain `clay` will run in the background.
+
+
+Install and use `crd`
```

Comment:
What is `crd`?

Reply:
The Camp Road Diner!

Sadly, it no longer exists and is now called Savory's.  Still one of the best breakfasts in my hometown ;).

https://www.savoryshamburg.com/

Reply:
LOL